### DISCUSSION TITLE
Awesome work

### DISCUSSION THREAD

**Discussion by Biliskn3r** [valinet/ExplorerPatcher#258] [General]
After a day of Windows 11 and the taskbar on the bottom, I nearly rolled back to Win10. On dual 27", a bottom taskbar is retarded.

But a bit of searching brought me here: so first of all, TYVM and GREAT WORK!

Feedback after 24 hours of use: alt-X is slow (5-8 second lag) when "enable start menu support" is ticked. Not just opening ExplorerPatcher, but anything in that menu (I use "Run" / WIN+R most often). I don't use any of the start-menu support options but thought you'd want the feedback.

Placing the taskbars to the sides and then turning off "Enable Windows 10 taskbar" breaks the whole thing (lmao Win11 dev team). No icons rendered, sometimes the start-menu shows up, the icons next to clock are overlapping each other (seriously Microsoft, what the hell?). I know your README says to place taskbar at the bottom before uninstalling, but perhaps a warning should come with unticking "enable Windows 10 taskbar" if you place your taskbar to the sides of the screen. I couldn't uninstall (the taskbar remained blank) and alt-x didn't have ExplorerPatcher to reset / re-enable "windows 10 taskbar" (I got around this by logging into a new user, grabbed default regkey settings, then imported them into broken user taskbar. So all good).

Keep up the great work. Taskbar to the sides of the screen is 110% must have. (I can live with Win11 right-click foobar for now).

**Comment by valinet** [comment]
Hi

First of all, thank you. Second of all, I presume you mean Win+X, not Alt+X, idk, for me it appears instantaneously, and works instantaneously as well, I am also testing on a laptop as well so not the most powerful machine out there. It’s one of the first features to have been implemented in this, I wouldn’t have left it in a state of taking 8 seconds for every action. Check to make sure other stuff is not interfering, experience tells that even the most unobvious stuff can still mess with things. I have a clean install without tons of other tweaks that I test on and it works just fine.

Regarding the taskbar, it’s clearly stated as a warning in the README. It’s the new taskbar’s fault that it tries to apply setting it does not support, really. Also, when faced with such a situation, easiest I think is to just reinstall EP, uninstall taking care to move the taskbar back as specified in the README, and then it should just work.

Anyway, thank you for the feedback.

**Reply by Biliskn3r** [reply]
Yes, Win-X! 

So when I moved the taskbar to the side, 'lock all taskbars', then turned off 'enable start menu support', it broke. And I couldn't move it back to bottom, bcos it was locked and right-clicking to untick "lock all taskbars" only showed the windows 11 (one option) menu. So I was stuck :)

I'll try to recreate the lag. Not sure what combination of options I was fiddling around with. Maybe it was registering the experimental DLLs.

**Reply by valinet** [reply]
Win+X is shown only when pressing Win+X or right clicking the Start button. Are you sure you are talking about that?  ...

**Reply by Biliskn3r** [reply]
Sorry to confuse, that was supposed to be two separate topics.

1. Let me test with the Win-X "lag" (it seems to be gone I'll try to recreate).
2. The taskbar stuck / locked-taskbar on the side of screen (a habit from Win10, locking my taskbars) was probably a rookie mistake.

**Reply by valinet** [reply]
Okay then. Maybe sometimes it takes longer, idk, but have never experienced that…

**Comment by ericblade** [comment]
I did not find a completely appropriate "APPRECIATION" thread, so I guess I'll make use of this one?

OMG Windows 11 is absolutely unusable without this patch.  

The 11 taskbar sucks compared to the 10 taskbar sucks compared to the 8 taskbar compared to the 7 taskbar .. but this at least makes it usable.  I've just used the hack for upgrading Insiders 10 to Insiders 11, so that I could get a preview of it (i tried a Surface Pro 3 that came with it, and it was an absolutely useless piece of garbage, before i even got to the software), and the machine was absolutely useless until I installed this patch.  

As soon as my finances are figured out for the next couple of months, I'm hitting the Sponsor button.

**Reply by valinet** [reply]
Thanks for the appreciation!

**Comment by RedDawgOne** [comment]
Thank you so much for this! I created this github account for the sole purposes of telling you how much this has saved my day. I had to get a new computer after my main one aged into oblivion, so I didn't have the option to roll back to 10. I always keep just five browser windows, separated, in the same order, with the first being email. Whenever I am done with a task I can just flick my eyes to the taskbar and see if I have a new email waiting. I have  20 years of habit working this way, and that's a hard history to rebuild. With the Windows 11 no-combine and no-label, I have missed so many emails and have spent so much time trying to figure out which window I need from dozens I have unnecessarily opened, that my job was legit in danger. From the bottom of my heart, thank you for building this and making it available.

**Reply by valinet** [reply]
I am super happy that it works for you. Thank you for the kind words! I as well would like to thank the community and the contributors that also helped and help build this bit by bit. Thanks!

Indeed, muscle memory and keeping your productivity is more important than any visual gimmick.
